src/status/notifications/osnotificationdetails.nim

```diff
@@ -0,0 +1,38 @@
+import json
```

Comment:
nit rename to: os_notification_details as we use underscore for file name in most of nim files